cli/csstools-cli/README.md

```diff
@@ -31,33 +31,51 @@ Usage:
   @csstools/csstools-cli postcss-preset-env <input.css>... [OPTIONS] --replace
 
 Available Plugins:
+  postcss-preset-env
   css-blank-pseudo
   css-has-pseudo
   css-prefers-color-scheme
+  postcss-attribute-case-insensitive
+  postcss-cascade-layers
   postcss-color-function
   postcss-color-functional-notation
   postcss-color-hex-alpha
+  postcss-color-mix-function
   postcss-color-rebeccapurple
+  postcss-custom-media
   postcss-custom-properties
+  postcss-custom-selectors
   postcss-dir-pseudo-class
   postcss-double-position-gradients
-  postcss-env-function
```

Comment:
This was already removed, only docs remained

Reply:
Nice catch